packages/flutter/lib/src/material/button_bar.dart

```diff
@@ -12,12 +13,18 @@ import 'raised_button.dart';
 /// An end-aligned row of buttons.
 ///
 /// Places the buttons horizontally according to the padding in the current
-/// [ButtonTheme]. The children are laid out in a [Row] with
+/// [ButtonBarTheme]. The children are laid out in a [Row] with
 /// [MainAxisAlignment.end]. When the [Directionality] is [TextDirection.ltr],
 /// the button bar's children are right justified and the last child becomes
 /// the rightmost child. When the [Directionality] [TextDirection.rtl] the
 /// children are left justified and the last child becomes the leftmost child.
 ///
+/// The [ButtonBar] can be configured with a [ButtonBarTheme]. For any property
+/// not set on the [ButtonBar] directly it will look for a surrounding
+/// [ButtonBarTheme] and use its properties. The button values in
+/// [ButtonBarTheme] will be used to override properties in the surrounding
```

Comment:
Since it's a little confusing, it might be worth devoting a short paragraph at the end here to the fact that ButtonBar wraps its child with a ButtonTheme whose ButtonThemeData is based on the button bar's parameters and the surrounding (sometimes we say "ambient") ButtonBarTheme.